ecs.tf

```diff
@@ -3,7 +3,7 @@ locals {
   container_definitions = [
     merge({
       "name" : module.label.id,
-      "image" : "${data.aws_ecr_repository.existing.repository_url}:${var.image_tag}",
+      "image" : "${var.external_image == "" ? data.aws_ecr_repository.existing[0].repository_url : var.external_image}:${var.image_tag}",
```

Comment:
What do you think about extracting this condition to the separate local variable?
e.g. `image_url = var.external_image == "" ? data.aws_ecr_repository.existing[0].repository_url : var.external_image`

Reply:
if this is the only instance where we use this I'd keep it as is